lib/msf/core/exploit/remote/ldap.rb

```diff
@@ -4,7 +4,7 @@
 # This mixin is a wrapper around Net::LDAP
 #
 
-require 'net-ldap'
+require 'rex/proto/ldap'
```

Comment:
This causes a warning for me on start up

```
$ bundle exec ./msfconsole -q
/mnt/hgfs/metasploit-framework/lib/rex/proto/ldap.rb:7: warning: already initialized constant Net::LDAP::Connection::LdapVersion
/var/lib/gems/2.7.0/gems/net-ldap-0.17.0/lib/net/ldap/connection.rb:9: warning: previous definition of LdapVersion was here
```

In the past we've foudn that warnings really confuse newer users, and they generally start raising issue reports as a result. Might be good to fix this 🕵️

Reply:
This should be resolved now. @adfoster-r7 / @zeroSteiner - could you please confirm?

Reply:
Confirmed this to have been resolved. :+1: